realm/realm-library/src/androidTest/java/io/realm/RealmQueryTest.java

```diff
@@ -55,15 +67,20 @@
     public final TestRealmConfigurationFactory configFactory = new TestRealmConfigurationFactory();
     @Rule
     public final ExpectedException thrown = ExpectedException.none();
+    @Rule
+    public final RunInLooperThread workerThread = new RunInLooperThread();
 
     protected final static int TEST_DATA_SIZE = 10;
 
+    private final static long DECADE_MILLIS = 10 * TimeUnit.DAYS.toMillis(365);
+
     protected Realm testRealm;
 
-    private final static long DECADE_MILLIS = 10 * TimeUnit.DAYS.toMillis(365);
+    private Context context;
 
     @Before
     public void setUp() throws Exception {
+        context = InstrumentationRegistry.getInstrumentation().getContext();
```

Comment:
You shouldn't have any need for the context. Everything should go through the configFactory